SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DefaultApplicationSettings.cs

```diff
@@ -137,5 +185,37 @@ private static Configuration OpenConfiguration()
             var executionFileMap = new ExeConfigurationFileMap { ExeConfigFilename = FilePath };
             return ConfigurationManager.OpenMappedExeConfiguration(executionFileMap, ConfigurationUserLevel.None);
         }
+
+        private void CreateConfigurationFileBackup()
+        {
+            try
+            {
+                if (!configuration.HasFile)
+                {
+                    configuration.Save(ConfigurationSaveMode.Full, true);
+                }
+                File.Copy(FilePath, BackupFilePath, true);
+            }
+            catch (ConfigurationErrorsException e)
+            {
+                AppCenterLog.Warn(AppCenterLog.LogTag, "Could not back up the configuration file.", e);
+            }
+        }
+
+        private bool RestoreConfigurationFile()
```

Comment:
Add XML doc to document returned value

Reply:
This function already reworked and its return type is 'void' for now.